package/src/components/Poll/components/CreatePollOptions.tsx

```diff
@@ -81,14 +94,19 @@ export const CreatePollOption = ({
   // used for swapping with newIndex
   const currentIndex = useSharedValue<number | null>(null);
 
+  // The sanity check for position cache updated index, is added because after a poll is sent its been reset
+  // by the composer and it throws an undefined error. This can be removed in future.
   useAnimatedReaction(
-    () => currentOptionPositionsDerived.value.positionCache[index].updatedIndex,
+    () =>
+      currentOptionPositionsDerived.value.positionCache[option.id]
+        ? currentOptionPositionsDerived.value.positionCache[option.id].updatedIndex
+        : 0,
```

Comment:
should this really default to `0` ?

Reply:
Also:

```
currentOptionPositionsDerived.value.positionCache[option.id]?.updatedIndex ?? 0
```

is probably a bit better

Reply:
I think 0 makes a sensible default here because worst case scenario we will always have an empty option from LLC that will be present. I will add the improvement.

Reply:
That is precisely why we do `?.updatedIndex` though. If you meant that `positionCache` can  be `undefined` somehow (which I don't think it can), the previous snippet also doesn't work.